source_gen/lib/src/builder.dart

```diff
@@ -106,13 +106,22 @@ class _Builder extends Builder {
                 'Consider adding the following to your source file:\n\n'
                 'library $suggest;');
       }
-      final part = computePartUrl(buildStep.inputId, outputId);
+      contentBuffer.writeln();
+
+      String part;
+      if (_outputPartOf) {
```

Comment:
Discussed offline. These variable names are confusing and not indicative in what we are using them for.

Reply:
right...but is this PR okay otherwise?

Reply:
I was hoping you'd fix it in this one.